File: samples/JavaScript/Json Store/Cursor/Select/By Full Text Search.js
session.setAsync(false);

var store = session.getDatabase("playground").getStore("forum");
var s = "";

s += ">>> Select all documents using full text search, 'yellow'\n";
var c = store.openCursor().range(0,5);

c.ftSearch('yellow').find(function(e) {
  s += "  title:"+e.getString("title")+"\n";
});  
s += "\n";


darwino.Utils.setText("content","{0}",s);

File: samples/Java/Json Store/Cursor/Select/By Full Text Search.java
Store store = session.getDatabase("playground").getStore("forum");
String s = ""; final StringBuilder b = new StringBuilder();

s += ">>> Select all documents using full text search, 'yellow'\n";
Cursor c = store.openCursor().range(0,5);

b.setLength(0);
c.ftSearch("yellow").find(new CursorHandler() {
  public boolean handle(CursorEntry e) throws JsonException {  
  	b.append("  title:"+e.getString("title")+"\n");
    return true;
  }
});  
s += b.toString()+"\n";


_formatText("{0}",s);